src/views/tunnel_setup.handlebars

```diff
@@ -19,6 +19,11 @@
       <p>Choose a secure web address for your gateway:</p>
       <input type="text" id="subdomain" placeholder="subdomain" required autofocus /><div id="label-subdomain">.{{domain}}</div>
       <input type="email" id="email" placeholder="Email" required /><br>
+      <div id="opt-in-group">
+        <input type="checkbox" id="opt-in" name="opt-in" checked>
+        <label for="opt-in"></label>
+        <span id="opt-in-info">Please send me more information about Mozilla IoT products from time to time.</span>
```

Comment:
I'm not sure this reflects the current/future emails we send people. @benfrancis ?

Reply:
Yeah, the wording can be changed easily. I wanted to go ahead and get the rest of the framework in place so that we just have to make an easy change.

Reply:
I'd suggest something like "Please keep me updated about new features and contribution opportunities." but we may need to get Legal to check the wording.

Reply:
I'm fine with keeping this PR open until we get wording figured out.

@benfrancis @hobinjk Are you both ok with the design and functionality?

Reply:
The UI looks fine :)

@faramoz Do we need to get someone in Legal to check this wording?

Reply:
I talked to Michael Feldman from legal yesterday about this and he would take a look at this. I need to copy this info and send it to him since I don't think he is in GitHub.